guidelines/releases/release-process.md

```diff
@@ -67,28 +87,35 @@ Follow these steps to create a release:
 2. Ensure that the `prow/RELEASE_VERSION` file from the `test-infra` repository on a release branch contains the correct version to be created. The file should contain a release version following the `{A}.{B}.{C}` or `{A}.{B}.{C}-rc{D}` format, where `A`,`B`, `C`, and `D` are numbers. If you define a release candidate version, a pre-release is created.  
 
     Make sure the `RELEASE_VERSION` file includes just this single line:  
-    ` echo -n $RELEASE_VERSION > prow/RELEASE_VERSION`
+    ```bash
+    echo -n {RELEASE_VERSION} > prow/RELEASE_VERSION
+    ```
 
-3. Push the branch to the `test-infra`  repository.
+3. Push the branch to the `test-infra` repository.
 
-4. Create a PR to `test-infra/release-x.y`. This triggers the pre-release job for watch-pods.
+4. Create a PR to `test-infra/release-x.y`. This triggers the pre-release job for `watch-pods`. 
+
+    > **NOTE:** If you want to just trigger `watch-pods` build without introducing any changes, edit any file within `test-infra` repository and create a pull request. You don't need to merge it - the job will be triggered anyway. After successful `watch-pods` image build, you can close the pull request.
+
+5. Prepare and merge pull request to `master` with updating RELEASE_VERSION for next minor release candidate. For example, if `RELEASE_VERSION` on `master` branch is set to `0.9.2`, then change the version to `1.0.0-rc1`.
```

Comment:
What do you mean by preparing the pull request? Is the `RELEASE_VERSION` a file? Would you agree with the below?
```suggestion
5. Update the `RELEASE_VERSION`file with the name of the next minor release candidate and merge the pull request to `master`. For example, if the `RELEASE_VERSION` on the `master` branch is set to `0.9.2`, then change the version to `1.0.0-rc1`.
```